<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Настройки Боксов</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_boxes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Привязка датчиков</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget_boxes_sensors"/>
         <widget class="QTableWidget" name="tableWidget_sensors"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Приём данных</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget_device"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_remove">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>881</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_addDevice">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_poll">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Таймер опроса</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Авария</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>238</width>
         <height>139</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Таймаут данных</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="time">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="value">
             <number>12</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_errorTempMax">
          <property name="text">
           <string>Превышение температуры</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_errorTempMin">
          <property name="text">
           <string>Занижение температуры</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_errorHumiMax">
          <property name="text">
           <string>Привышени влажности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_errorHumiMin">
          <property name="text">
           <string>Занижение влажности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_errorData">
          <property name="text">
           <string>Отсутствствие данных</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Безопастность</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>155</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>Новый пароль..</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPassword">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Ок</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Пути</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Папка отчётов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pathReport">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pathReport">
           <property name="text">
            <string>Обзор..</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>630</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
